lib/coverage.js

```diff
@@ -270,7 +270,7 @@ internals.instrument = function (filename) {
         ecmaFeatures: {
```

Comment:
It's worth verifying, but I think this `ecmaFeatures` config can go away if we bump the `ecmaVersion` to `9`.

Reply:
It looks like the option is removed in espree v4.0.0.alpha.0 https://github.com/eslint/espree/pull/374